pipe_anchorages/transforms/voyages_create.py

```diff
@@ -0,0 +1,62 @@
+import apache_beam as beam
+import datetime as dt
+import pytz
+
+def create_voyage(visits):
+    # first voyage start from 'we do not know' = None
+    visit_before = {2: None, 3: None, 4: None}
+
+    def min_confidence(v, c:int)->bool:
+        return v['confidence'] >= c
```

Comment:
I think this function is too much of an abstraction. I was reading below where it's used and I couldn't figure out what it's supposed to mean. It's clearer to me if I see this:

```python
if (visit and visit["confidence"] >= c and has_duration(visit_before[c], visit)) or (not visit and visit_before[c])):
```